src/awesome_o/state.clj

```diff
@@ -21,17 +21,22 @@
 (defn reset-state []
   (wcar* (car/set "state"
                   {:persons {}
-                   :locations []
-                   :slackmasters-index 0
-                   :dev-meeting-index 0})))
+                   :task-assignments {:slackmaster nil
+                                      :meetingmaster nil}})))
```

Comment:
Aren't you glad that:

```clojure
(update-in nil [:a :b] :c "hello")
;;=> {:a {:b {:c "hello"}}}
```
? :) which means we don't need to "migrate" the DB there, just select the next slack master